components/caption_images/src/main.py

```diff
@@ -12,75 +13,97 @@
 
 logger = logging.getLogger(__name__)
 
+os.environ['CUDA_LAUNCH_BLOCKING'] = "1"
```

Comment:
We no longer need this, right? It was just for debugging I think.
Same in the other components.

Reply:
Kept it in case we run into other issues later on and we need further debugging. We can remove it later on once we've tested enough GPU components at scale and are sure that everything runs fine

Reply:
I don't think it helped us since are already running single-threaded. It also didn't change the stacktrace, the first one was already correct. And it seems like it [really should not be used when not debugging](https://forums.developer.nvidia.com/t/cuda-graph-api-fail-when-cuda-launch-blocking-1-is-set/265619/2).

Reply:
oh ok, I'll remove it then